pkgs/top-level/all-packages.nix

```diff
@@ -12944,7 +12944,7 @@ in
 
   herqq = libsForQt514.callPackage ../development/libraries/herqq { };
 
-  hidapi = callPackage ../development/libraries/hidapi { };
+  hidapi = callPackage ../development/libraries/hidapi { systemd = systemdMinimal; };
```

Comment:
does systemd now depend on this somehow, or is this a leftover from the homed PR (and some of the dependencies that might have spilled over and caused this (possible) bootstrap problem)?

Reply:
I think it's a left over, but should not harm. Should I still delete it?

Reply:
IIRC its a dependecy of libfido2, which is needed for homed. So I removed it